src/main/scanImages.ts

```diff
@@ -22,6 +22,9 @@ export function scanImagesFactory(serviceContainer: LocalMain.ServiceContainerSe
 			const imageData = siteData.imageData || {};
 
 			if (!site) {
+				/**
+				 * @todo does passing the error object here translate correctly to the UI?
```

Comment:
Decided not to fix this up here so as to not bloat this PR. @tihal what do you think of grouping this in with: https://getflywheel.atlassian.net/browse/LOC-2276?

Reply:
For MLP we decided to just go with reporting errors to Local Log, so there's nowhere that those errors could get output to the UI at this time.

We could either:
1 - consider updating our designs for MLP to output errors on the UI
2 - make sure we get a follow on story written that would include this work, as well as updates to the UI